app/actions/index.js

```diff
@@ -276,3 +280,46 @@ export const listRemoteSurveys = () => (dispatch, getState) => {
       );
   });
 };
+
+export const initializeObservation = tags => dispatch =>
+  dispatch({
+    type: types.INITIALIZE_OBSERVATION,
+    tags
+  });
+
+export const updateObservation = tags => dispatch =>
+  dispatch({
+    type: types.UPDATE_OBSERVATION,
+    tags
+  });
+
+export const saveObservation = () => (dispatch, getState) => {
+  const { observation } = selectActiveObservation(getState());
+
+  dispatch({
+    type: types.SAVING_OBSERVATION,
+    observation
+  });
+
+  // TODO this is wrong
+  // observation currently looks like (still needs a point / associated feature):
+  // {
+  //   tags: {
+  //     highway: "residential",
+  //     name: "My Street",
+  //     surface: "paved"
+  //   }
+  // }
+  return osm.createObservation(observation, error => {
```

Comment:
Hey @mojodna,

[osm-p2p-observations](https://github.com/digidem/osm-p2p-observations/) expects an [OSM node document](https://wiki.openstreetmap.org/wiki/API_v0.6#Create:_PUT_.2Fapi.2F0.6.2F.5Bnode.7Cway.7Crelation.5D.2Fcreate) but in JSON form, with `type` set to `"observation"`. So something like

```js
{
   type: "observation",
   lat: 100,
   lon: -50,
   tags: {
     highway: "residential",
     name: "My Street",
     surface: "paved"
   }
}
```

+1 on ensuring we include `deviceId`. As well as `presetId` and `timestamp`.